website/core/Marked.js

```diff
@@ -595,6 +595,13 @@ InlineLexer.prototype.output = function(src) {
     // tag
     if (cap = this.rules.tag.exec(src)) {
       src = src.substring(cap[0].length);
+
+      var color = cap[0].match('<color ([^ ]+) />');
```

Comment:
This is a bit annoying to have to modify Marked.js to add it but it wasn't easy to support custom html tags. Marked outputs '<span>' and '</span>' as different tokens, and not one "ast-node" that we can just convert to React.DOM like the other things here